package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "glue",
   "description": "Server composer for hapi.js",
-  "version": "4.1.0",
+  "version": "4.1.1",
```

Comment:
@csrl I will revert does changes and make a new pull request. I will let you know about this tonight.